Add icingaweb2-module-perfdatagraphs and its backends (#11)
Donien/ansible-collection-icinga_contrib: roles/icingaweb2_modules/README.md

    `config.graphs` **takes precedence** over the \*.ini files when redefining the same section.  
    Default: `false`
  - `config.grafana`: `dictionary`  

>     Manages *config.ini* to configure the module. Its keys are equal to the modules [configuration file](https://github.com/netwaays/icingaweb2-module-grafana/blob/master/doc/03-module-configuration.md#example-configini-etcicingaweb2modulesgrafanaconfigini), though not necessarily complete yet.  

Reviewer: ```suggestion
    Manages *config.ini* to configure the module. Its keys are equal to the modules [configuration file](https://github.com/NETWAYS/icingaweb2-module-grafana/blob/main/doc/03-module-configuration.md#available-options), though not necessarily complete yet.  
```

    For possible default values have look at [templates/grafana/config.ini.j2](templates/grafana/config.ini.j2).
  - `config.graphs`  

>     Manages *graphs.ini* to configure the module. Its keys are equal to the modules [graphs file](https://github.com/netways/icingaweb2-module-grafana/blob/master/doc/04-graph-configuration.md#options), though not necessarily complete yet.  

Reviewer: ```suggestion
    Manages *graphs.ini* to configure the module. Its keys are equal to the modules [graphs file](https://github.com/NETWAYS/icingaweb2-module-grafana/blob/master/doc/04-graph-configuration.md#options), though not necessarily complete yet.  
```

- `icingaweb2_modules.mapDatatype`: `none`  
  - No keys for this module.

## perfdatagraphs

- `icingaweb2_modules.perfdatagraphs`: `directory`
  - `config.perfdatagraphs`: `dictionary`  

>     Manages the Section perfdatagraphs within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs/blob/main/doc/03-Configuration.md).  

Reviewer: ```suggestion
    Manages the section perfdatagraphs within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs/blob/main/doc/03-Configuration.md).  
```

    For possible default values have look at [templates/perfdatagraphs/config.ini.j2](templates/perfdatagraphs/config.ini.j2).

## perfdatagraphsgraphite

- `icingaweb2_modules.perfdatagraphsgraphite`: `directory`
  - `config.perfdatagraphsgraphite`: `dictionary`  

>     Manages the Section perfdatagraphsgraphite within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs-graphite/tree/main/doc).  

Reviewer: ```suggestion
    Manages the section perfdatagraphsgraphite within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs-graphite/tree/main/doc).  
```

    For possible default values have look at [templates/perfdatagraphsgraphite/config.ini.j2](templates/perfdatagraphsgraphite/config.ini.j2).

## perfdatagraphsinfluxdbv1

- `icingaweb2_modules.perfdatagraphsinfluxdbv1`: `directory`
  - `config.perfdatagraphsinfluxdbv1`: `dictionary`  

>     Manages the Section perfdatagraphsinfluxdbv1 within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs-influxdbv1/tree/development/doc).  

Reviewer: ```suggestion
    Manages the section perfdatagraphsinfluxdbv1 within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs-influxdbv1/tree/development/doc).  
```

    For possible default values have look at [templates/perfdatagraphsinfluxdbv1/config.ini.j2](templates/perfdatagraphsinfluxdbv1/config.ini.j2).

## perfdatagraphsinfluxdbv2

- `icingaweb2_modules.perfdatagraphsinfluxdbv2`: `directory`
  - `config.perfdatagraphsinfluxdbv2`: `dictionary`  

>     Manages the Section perfdatagraphsinfluxdbv2 within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs-influxdbv2/tree/main/doc).  

Reviewer: ```suggestion
    Manages the section perfdatagraphsinfluxdbv2 within *config.ini* to configure the module. Its keys are equal to the module's [settings](https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs-influxdbv2/tree/main/doc).  
```

        # Avoid removing Grafana as in "the Grafana server"
        package_name: "icinga-grafana"
        source: git

>         url: "https://github.com/netways/icingaweb2-module-grafana.git"

Reviewer: ```suggestion
        url: "https://github.com/NETWAYS/icingaweb2-module-grafana.git"
```

        # Avoid removing Grafana as in "the Grafana server"
        package_name: "icinga-grafana"
        source: git

>         url: "https://github.com/netways/icingaweb2-module-grafana.git"

Reviewer: ```suggestion
        url: "https://github.com/NETWAYS/icingaweb2-module-grafana.git"
```

        source: git
        url: "https://github.com/nbuchwitz/icingaweb2-module-mapDatatype.git"
```

## perfdatagraphs with Backend Playbooks

Install the perfdatagraphs module via Git.

```yaml
- name: Manage perfdatagraphs module
  hosts:
    - icingaweb2

  vars:
    icingaweb2_modules:
      perfdatagraphs:
        package_name: "icinga-perfdatagraphs"
        source: git

>         url: "https://github.com/netways/icingaweb2-module-perfdatagraphs"

Reviewer: ```suggestion
        url: "https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs"
```

        enabled: true
        config:
          perfdatagraphs:
            default_backend: InfluxDBv2
      perfdatagraphsinfluxdbv2:
        package_name: "icinga-perfdatagraphs-influxdbv2"
        source: git

>         url: "https://github.com/netways/icingaweb2-module-perfdatagraphs"

Reviewer: ```suggestion
        url: "https://github.com/NETWAYS/icingaweb2-module-perfdatagraphs"
```
